src/shells/zsh.rs

```diff
@@ -37,7 +37,16 @@ pazi_cd() {
     esac
 }
 alias z='pazi_cd'
-"#
+
+_pazi_cd() {
+  CURRENTWORD="${LBUFFER/* /}${RBUFFER/ */}"
+  local suggestions=(${(f)"$(pazi complete zsh $CURRENTWORD)"})
+  _describe -t pazi-dirs 'pazi' suggestions
+}
+
+compdef _pazi_cd pazi_cd 'pazi jump'
```

Comment:
One final thing: I think that this should be wrapped in `if whence compdef>/dev/null; then ....`

There are some distros that don't ship with compdef on (e.g. debian, tested in the default docker image at least).

In those distros, we should just not turn on completions  rather than  print out errors on  startup I think.

Reply:
That's fair. Thanks